plugins/example-code-snippets/src/api-support/api-support-matrix-generator.ts

```diff
@@ -235,7 +235,15 @@ const TOPIC_API_GROUPS: Array<ApiGroup> = [
   {
     groupName: 'Topics',
     groupDescription: 'A matrix of SDK support for Momento Topics APIs',
-    apis: ['subscribe', 'publish'],
+    apis: [
+      'subscribe',
+      'publish',
+      'listWebhooks',
+      'putWebhook',
+      'deleteWebhook',
+      'getWebhookSecret',
+      'rotateWebhookSecret',
+    ],
```

Comment:
It would be awesome to also add your `Webhooks.validateRequestSignature` thing too so that it is visible on these pages.

I know it wouldn't really fit here without some additional hackery, so it doesn't need to block this PR, but would like to know what you think about maybe trying to do something as a follow-up PR.